app/routes/admin/groups/components/AddGroupMember.js

```diff
@@ -1,14 +1,16 @@
 // @flow
 import React from 'react';
 import { Field, SubmissionError } from 'redux-form';
-import type { FieldProps } from 'redux-form';
+import type { FormProps } from 'redux-form';
 import { legoForm, Button, Form } from 'app/components/Form';
 import SelectInput from 'app/components/Form/SelectInput';
 import { createValidator, required } from 'app/utils/validation';
 import { ROLES } from 'app/utils/constants';
+import type { role } from 'app/models';
 
-type Props = FieldProps & {
-  group: Object
+type Props = FormProps & {
+  group: Object,
+  addMember: ({ role: role, userId: Number, groupId: Number }) => Promise<*>
```

Comment:
```suggestion
  addMember: ({ role: $Keys<typeof ROLES>, userId: Number, groupId: Number }) => Promise<*>
```

Reply:
The `role` type is the same as `$Values<typeof ROLES>` and not the key (the value we send to the api).

Reply:
Thanks! Always nice to get feedback from people who know what they are doing.